zulipterminal/model.py

```diff
@@ -555,6 +559,22 @@ def _stream_info_from_subscriptions(
                    key=lambda s: s[0].lower())
         )
 
+    def _group_info_from_realm_user_groups(self,
+                                           groups: List[Dict[str, Any]]
+                                           ) -> List[str]:
+        """
+        Stores group information in the index and returns a list of
+        group_names which helps in group typeahead. (Eg: @*terminal*)
+        """
+        for sub_group in groups:
+            self.index['groups'][sub_group['id']] = {
+                key: sub_group[key] for key in sub_group if key != 'id'}
+        user_group_list = [self.index['groups'][group_id]['name']
+                           for group_id in self.index['groups']]
+        # Sort groups for typeahead to work alphabetically
+        sorted(user_group_list)
```

Comment:
I'm pretty sure this doesn't quite do what you expect; an unsorted result would show this not working.

Reply:
Oops. I should have assigned it back. Or better to use `.sort()`